src/dataframerow/dataframerow.jl

```diff
@@ -1,85 +1,117 @@
 """
-    DataFrameRow{<:AbstractDataFrame}
+    DataFrameRow{<:AbstractDataFrame,<:AbstractIndex}
 
-A view of one row of an AbstractDataFrame.
+A view of one row of an `AbstractDataFrame`.
+
+A `DataFrameRow` is constructed with `view` or `getindex` when one row and a
+selection of columns are requested, or when iterating the result
+of the call to the [`eachrow`](@ref) function.
+
+### Arguments
```

Comment:
This doesn't apply here either.

Reply:
fixed - but please review - I left the comment above unresolved to keep track of this.